server/zanata-war/src/test/java/org/zanata/ZanataRestTest.java

```diff
@@ -79,12 +84,11 @@ public final void prepareRestEasyFramework() {
                     providerInstance);
         }
 
-        InMemoryClientExecutor executor =
-                new InMemoryClientExecutor(dispatcher);
```

Comment:
@aeng I've been looking at this today, but I'm not a fan of suppressing warnings unless there is no other alternative.

I want to look at these tests before we delete or disable the mocking/in-memory functionality. I think InMemoryClientExecutor is deprecated, isn't it? Right now I'm not sure if there's a non-deprecated alternative.